7/debian-10/rootfs/opt/bitnami/scripts/libkibana.sh

```diff
@@ -181,8 +199,9 @@ is_kibana_not_running() {
 #   Boolean
 #########################
 is_kibana_ready() {
+    local basePath=$(kibana_conf_get "[server.basePath]")
     if is_kibana_running; then
-        local -r state="$(yq r - "status.overall.state" <<<"$(curl -s "127.0.0.1:${KIBANA_PORT_NUMBER}/api/status")")"
+        local -r state="$(yq r - "status.overall.state" <<<"$(curl -s "127.0.0.1:${KIBANA_PORT_NUMBER}${basePath}/api/status")")"
```

Comment:
Hi @jerryharrow,

As discussed here https://github.com/bitnami/bitnami-docker-kibana/issues/48#issuecomment-847134835, this has to be conditional depending on the value of `server.rewriteBasePath` in the Kibana configuration.

According to Kibana documentation, the default value is `true`, so I'd recommend doing the following:
```suggestion
     local basePath=$(kibana_conf_get "[server.basePath]")
     local rewriteBasePath=$(kibana_conf_get "[server.basePath]")
     if is_kibana_running; then
         if [[ "$rewriteBasePath" == "false" ]]; then
             local -r state="$(yq r - "status.overall.state" <<<"$(curl -s "127.0.0.1:${KIBANA_PORT_NUMBER}/api/status")")"
         else
             local -r state="$(yq r - "status.overall.state" <<<"$(curl -s "127.0.0.1:${KIBANA_PORT_NUMBER}${basePath}/api/status")")"
         fi
```

Reply:
Added check to rewriteBasePath.